src/commands.rs

```diff
@@ -45,34 +47,32 @@ pub async fn preview(
     ctx: PoiseContext<'_>,
     #[description = "The preview you want to see"]
     #[autocomplete = "autocomplete_name"]
-    choice: String, ) -> Result<(), Error> {
-
-    if !CONFIG.colors.contains_key(choice.as_str()) {
+    choice: String,
+) -> Result<(), Error> {
+    if let Some(color) = CONFIG.colors.get(&choice) {
+        let paths = [CreateAttachment::bytes(create_image(&color), format!("{choice}.png"))];
+        ctx.send(CreateReply {
+            content: Some("Preview:".into()),
+            attachments: paths.to_vec().into(),
+            ..Default::default()
+        })
+        .await?;
+    } else {
```

Comment:
why not `match` to be consistent with the function above if you’re handling both cases anyway?